build_library/extra_sysexts.sh

```diff
@@ -2,4 +2,24 @@ EXTRA_SYSEXTS=(
   zfs:sys-fs/zfs
   podman:app-containers/podman,net-misc/passt
   python:dev-lang/python,dev-python/pip
+  nvidia-drivers-535:=x11-drivers/nvidia-drivers-535.230.02:-kernel-open
+  nvidia-drivers-535-open:=x11-drivers/nvidia-drivers-535.230.02:kernel-open
+  nvidia-drivers-550:=x11-drivers/nvidia-drivers-550.144.03:-kernel-open
+  nvidia-drivers-550-open:=x11-drivers/nvidia-drivers-550.144.03:kernel-open
```

Comment:
We could possibly make maintenance easier if we refer to these by SLOT instead, e.g. `nvidia-drivers-535|x11-drivers/nvidia-drivers:0/535|-kernel-open`. The `emerge` command will understand that, although I've had to change the separator to `|`.